cpp/doxygen/Doxyfile

```diff
@@ -1170,7 +1206,7 @@ HTML_COLORSTYLE_SAT    = 255
 # Minimum value: 40, maximum value: 240, default value: 80.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_COLORSTYLE_GAMMA  = 80
+HTML_COLORSTYLE_GAMMA  = 52
```

Comment:
How did you get the `HTML_COLORSTYLE_HUE`  and `HTML_COLORSTYLE_GAMMA`  values?

Reply:
I started by using an online Hex->HSV/HSB converter. Then I tried the values it gave me and set Gamma at 100 (no adjustment). Then I started adjusting gamma up and down to see how it affected Doxygen's output hex colors for table headers (e.g. the one in one of your markdown pages) since that is one of the only places Doxygen uses full-strength colors until I got that generated hex color to be nearly the same as the RAPIDS menu. It was trial and error, but I couldn't come up with another way to figure out how Doxygen goes from these settings to hex colors short of digging through its source.